env.yml

```diff
@@ -3,6 +3,7 @@ channels:
   - conda-forge
 dependencies:
   - clang=14.0.6
+  - clang-tools=14.0.6
```

Comment:
And clang-format?? I don't know, but guessing yes.

Reply:
Not needed - clang-tools has a dependency on it, so it gets installed automatically:
```
clang-tools 14.0.6 default_hf5194b7_0
-------------------------------------
file name   : clang-tools-14.0.6-default_hf5194b7_0.conda
name        : clang-tools
version     : 14.0.6
build       : default_hf5194b7_0
build number: 0
size        : 9.8 MB
license     : Apache-2.0 WITH LLVM-exception
subdir      : osx-arm64
url         : https://repo.anaconda.com/pkgs/main/osx-arm64/clang-tools-14.0.6-default_hf5194b7_0.conda
md5         : 94cac2f8bc7b1579595d258bde3bde83
timestamp   : 2022-08-31 06:32:05 UTC
constraints : 
  - clangdev 14.0.6
dependencies: 
  - clang-format 14.0.6 default_hf5194b7_0
  - libclang >=14.0.6,<14.1.0a0
  - libclang-cpp14 >=14.0.6,<14.1.0a0
  - libcxx >=12.0.0
  - libllvm14 >=14.0.6,<14.1.0a0
```